gegelatilib/include/learn/adversarialLearningAgent.h

```diff
@@ -65,6 +74,8 @@ namespace Learn {
          * This method gathers results in a map linking root to result, and
          * then reverts the map to match the "results" argument.
          * The archive will just be merged like in ParallelLearningAgent.
+         * Note that if there is a "posOfStudiedRoot" in the jobs, we will
+         * only take the score of the given root and others will be ignored.
```

Comment:
So, "only the EvaluationResult of the posOfStudiedRoot will be written to the results map. And the results of the other roots within the Job will be discarded"? (If yes, please complete the comment)